cmd/kola/spawn.go

```diff
@@ -125,10 +130,31 @@ func doSpawn(cmd *cobra.Command, args []string) error {
 		defer cluster.Destroy()
 	}
 
+	var updateConf *strings.Reader
+	if spawnOmahaPackage != "" {
+		qc, ok := cluster.(*qemu.Cluster)
+		if !ok {
+			//TODO(lucab): expand platform support
+			return errors.New("--omaha-package is currently only supported on qemu")
```

Comment:
Why use `errors.New` vs `fmt.Errorf` like the `spawnNodeCount` check?

Reply:
Because there is no string formatting involved, error message is constant.